packages/preact/index.js

```diff
@@ -53,4 +54,12 @@ module.exports = (neutrino, opts = {}) => {
         .set('react-dom', 'preact-compat')
         .set('create-react-class', 'preact-compat/lib/create-react-class')
         .set('react-addons-css-transition-group', 'preact-css-transition-group');
+
+  neutrino.config.when(neutrino.config.module.rules.has('lint'), () => {
```

Comment:
Was this change related to the rest of the PR? I couldn't see anything explaining it in the commit message or review comments? 

Also could you add a comment explaining why this is needed? :-)

Reply:
This makes it so we can use JSX without having to manually say:

```js
import React from 'react';
```

This was supposed to be part of the create-project fixes, but I did that work in tandem with this one because they were interlinked, and I tried to split them out, but unfortunately missed a few things.